nixos/modules/services/torrent/transmission.nix

```diff
@@ -288,13 +288,14 @@ in
         '';
       };
 
+
     systemd.services.transmission = {
       description = "Transmission BitTorrent Service";
       after = [ "network.target" ] ++ optional apparmor.enable "apparmor.service";
       requires = optional apparmor.enable "apparmor.service";
       wantedBy = [ "multi-user.target" ];
       environment.CURL_CA_BUNDLE = etc."ssl/certs/ca-certificates.crt".source;
-      environment.TRANSMISSION_WEB_HOME = lib.optionalString (cfg.webHome != null) cfg.webHome;
+      environment.TRANSMISSION_WEB_HOME = cfg.webHome;
```

Comment:
Suggestion: what if you used `lib.mkIf` instead of `lib.optionalString`? That would avoid having to change the `default` (which would need a `defaultText`).

Reply:
Will it unset the environment variable if the value is `null`?

Reply:
I tested. It works!